lib/bolt/config.rb

```diff
@@ -338,9 +338,17 @@ def deep_clone
 
     private def update_logs(logs)
       logs.each_with_object({}) do |(key, val), acc|
-        next unless val.is_a?(Hash)
+        # Remove any disabled logs
+        next if val == 'disable'
```

Comment:
Do we want to allow users to disable `console` as well? The [schema](https://github.com/puppetlabs/bolt/pull/2121/files#diff-893a47fc3cbeb564a562c04c9348ae96R185) currently says that it only permits a hash.

Reply:
I started to make that update when I was adding the `enum` property for other logs. It's complicated because `--log-level` turns into a log entry that will override `disable`. I'm not sure we want that.

Right now you _can't_ disable console, because `Bolt::Logger` will define it regardless of whether you specify it in your config or not. I want to move away from using the console as a log destination entirely, so I'm not super concerned about allowing it to be disabled now. The only thing you could really accomplish with that would be to silence warnings and errors, which seems unwanted anyway.